groups/infrastructure/profiles/live-eu-west-2/live/vars

```diff
@@ -0,0 +1,14 @@
+aws_bucket = "live-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-service-live-terraform-state"
+environment = "live"
+deploy_to = "live"
+state_prefix = "env:/live"
```

Comment:
The legacy remote_state vars should not be required and should be removed.
- `aws_bucket`
- `remote_state_bucket`
- `deploy_to`
- `state_prefix`

Reply:
This is fixed now. Commit is pushed.